File: Java/Framework/src/metrics/performance/Time.java
package metrics.performance;

abstract class Time {
    protected long start;
    protected long finish;

    public void startTimer(){
        start = System.currentTimeMillis();
    }
    public void endTimer(){
        finish = System.currentTimeMillis();
    }

    public long getResponseTime(){
        return finish - start;
    }
}

File: Old_Experiments/Dart/Framework/metrics/performance/time.dart
abstract class Time {
  late int start;
  late int finish;

  void startTimer() {
    start = DateTime.now().millisecondsSinceEpoch;
  }

  void endTimer() {
    finish = DateTime.now().millisecondsSinceEpoch;
  }

  int getResponseTime() {
    return finish - start;
  }
}
